applications/imaging_ai_segmentator/README.md

```diff
@@ -1,198 +1,175 @@
 ```
 
-**_Note_**
-It takes quite a few minutes when this command is run the first time.
-
-### Run the Application in Dev Environment
-
-It is strongly recommended a Python virtual environment is used for running the application in dev environment.
-
-This application only has Python implementation depending on a set of Python packages from [Pypi](https://pypi.org), however, a `build_and_install` step is needed to automate organizing Python code and downloading the model.
+You can modify them by setting the right env variables and mount the right volumes, for instance:
 
-
-Set up the Holohub environment, if not already done
 ```bash
-./run setup
+./dev_container build_and_run imaging_ai_segmentator --container_args "-v /local/output:/my_output -e HOLOSCAN_OUTPUT_PATH=/my_output"
 ```
 
-Set the environment variables for the application
+By default, the application uses the following directories for input data, model files, and output results:
+
 ```bash
-source applications/imaging_ai_segmentator/env_settings.sh
+HOLOSCAN_INPUT_PATH=<LOCAL_HOLOHUB_PATH>/data/imaging_ai_segmentator/dicom
+HOLOSCAN_MODEL_PATH=<LOCAL_HOLOHUB_PATH>/data/imaging_ai_segmentator/models
+HOLOSCAN_OUTPUT_PATH=<LOCAL_HOLOHUB_PATH>/build/imaging_ai_segmentator/output
 ```
 
-If not already done, download images of a CT series from [TCIA](https://nbia.cancerimagingarchive.net/nbia-search/), unzip if necessary, and save the folder of DICOM files under the folder `$HOLOSCAN_INPUT_PATH`.
+### Development Environment Setup
 
-Optionally download the AI model from [MONAI Model Zoo](https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBody_ct_segmentation), or wait till the build step to have it downloaded automatically
+You can run the application either in your local development environment or inside the Holohub development container.
 
-```bash
-mkdir -p $HOLOSCAN_MODEL_PATH
-pip install gdown
-python -m gdown https://drive.google.com/uc?id=1PHpFWboimEXmMSe2vBra6T8SaCMC2SHT -O $HOLOSCAN_MODEL_PATH/model.pt
-```
+1. **Set up the Holohub environment:**
 
-Install Python packages required by the application
-```bash
-pip install -r applications/imaging_ai_segmentator/requirements.txt
-```
+   A. **Within Container:**
 
-Build and install the application
-```bash
-./dev_container build_and_install imaging_ai_segmentator
-```
+   - Build and launch the Holohub Container:
 
-Run the application
-```bash
-rm -f -r $HOLOSCAN_OUTPUT_PATH
-python install/imaging_ai_segmentator/app.py
-```
+      ```bash
+      ./holohub run-container imaging_ai_segmentator
+      ```
 
-**_Note_**
-If desired, run the application with explicitly input, output, and/or model folder path, for example
-```bash
-rm -f -r ./output
-python install/imaging_ai_segmentator/app.py -m $HOLOSCAN_MODEL_PATH -i $HOLOSCAN_INPUT_PATH -o ./output
-```
+   B. **On Bare Metal (not recommended):**
 
-Check output
-```bash
-ls $HOLOSCAN_OUTPUT_PATH
-```
+    - Set up the Holohub environment:
```

Comment:
Please bring back the important [precaution](https://github.com/nvidia-holoscan/holohub/blob/main/applications/imaging_ai_segmentator/README.md#run-the-application-in-dev-environment) to use virtual env.